assets/stylesheets/_components.scss

```diff
@@ -158,7 +158,7 @@
 @import 'components/forms/form-text-input-with-affixes/style';
 @import 'components/forms/form-toggle/style';
 @import 'components/forms/range/style';
-@import 'components/forms/sortable-list/index';
+@import 'components/forms/sortable-list/index.scss';
```

Comment:
fine with me!

Reply:
on second thought there are also a number that can't be renamed `style` because they exist in the same directory.

I don't want to avoid adding `.scss` either - I want them to eventually all have the extension with them just as a way of removing "magic" from the system (removing implicit behaviors that aren't obvious until you learn the system when in fact there was no need to confusing)

this PR is mainly trying to be a very minimal change which allows us to build with the CSS loaders, the conflicting module names being a blocker.

I'll update the ones that make sense and leave the others, still trying to preserve the minimal changeset

Reply:
Torn on this one, since you both bring up good points.

Arguably, our current convention is `style.scss`, so `index.scss` files can be considered not following that convention. Also, renaming _one_ `style.jsx` file to something else seems easy enough.

So given our current convention, I'd agree that 

```scss
@import 'components/forms/sortable-list/style';
```

would be nicer that

```scss
@import 'components/forms/sortable-list/index.scss';
```

(Updating that convention to use `index.scss` is a separate concern IMO. Plus, it could lead to even more collisions, somewhat related to discussions we had in previous PRs. E.g. what will `import 'components/forms/sortable-list'` do? Import the js? The scss? Will it depend on context? Would this force us to be fully explicit, i.e `import 'components/forms/sortable-list/index.js'`? Let's not go there, at least for now :sweat_smile: )